modules/tinymce/src/plugins/visualchars/main/ts/core/Bindings.ts

```diff
@@ -5,16 +5,15 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
+import { Cell } from '@ephox/katamari';
 import Editor from 'tinymce/core/api/Editor';
 import * as Settings from '../api/Settings';
 import * as Actions from './Actions';
 
-const setup = (editor: Editor, toggleState) => {
+const setup = (editor: Editor, toggleState: Cell<boolean>) => {
   editor.on('init', () => {
-    // should be false when enabled, so toggling will change it to true
-    const valueForToggling = !Settings.isEnabledByDefault(editor);
-    toggleState.set(valueForToggling);
-    Actions.toggleVisualChars(editor, toggleState);
+    toggleState.set(Settings.isEnabledByDefault(editor));
```

Comment:
Can we move this to `Plugin.ts` instead, so it's where the cell originally gets initialized?